dev-libs/leveldb/metadata.xml

```diff
@@ -2,6 +2,9 @@
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
   <!-- maintainer-needed -->
+  <use>
+    <flag name="crc32c">Enable accelerated crc32 computation using <pkg>dev-libs/crc32c</pkg></flag>
```

Comment:
Any reason not to enable that unconditionally? It sounds a bit like `USE=-crc32c` uses bundled CRC32 routines.

Reply:
Was thinking it as an easier mask if there would be a need for an arch around `dev-libs/crc32c`. I saw that there is a basic, not hardware accelerated version of crc32c, bundled inside leveldb.

But after extra investigation of `dev-libs/crc32c`, I found it also has the same portable code, meaning it should work on other arches without big hiccups (I hope), after the required keywording.

Fixed this point.